templatetags/high_health_helpers.py

```diff
@@ -16,11 +16,14 @@ def goal_format(measure):
     if measure.metric.frequency == "MoM":
         current_month = measure.month
         last_year = measure.year - 1
-        previous_outcome = Measure.objects.filter(
-            metric=measure.metric.id,
-            date__month=current_month,
-            date__year=last_year,
-            school=measure.school)[0].value
+        try:
```

Comment:
Is it worth explaining in a comment why we need this for knowledge management for yourself?

Reply:
I do. I'll add one tomorrow.